### PR TITLE
Runo: Sets up CircleCI 

### PR DESCRIPTION
**Pull request by Runo-saduwa**
#### What does this PR achieve?
- Sets up continuous integration with circleci
- Sets up database image in circleci config
- Installs cross-env to account for cross-plaform compatability
- ensures all builds passes

#### Manual Testing Steps
- fetch this branch 
- run `git checkout ch-runo-circleci_config`
- Run `npm run test` to run tests locally 
- Run `git push`
- check [jobs in circleci] (https://circleci.com/gh/chidioguejiofor/) to see build results

#### Relevant Trello Tickets
- [Set Circle Ci and coveralls, code-climate](https://trello.com/c/Aea2u7l7/9-set-circle-ci-and-coveralls-code-climate)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                TEST_DB_USER: my_user
+                TEST_DB_NAME: my_test_db
+            #   environment:
+            #     TEST_DATABASE_URL: postgresql://root@localhost:8080/volunteersngtest
+            - image: postgres:9.6.2-alpine
+              environment:
+                POSTGRES_USER: my_user
+                POSTGRES_DB: my_test_db
+
+            # Specify service dependencies here if necessary
```
Comment (chidioguejiofor) [inline .circleci/config.yml]: Take out all the unnecessary comments in this file

File: .env-sample [hunk 2]
```diff
@@ -1,4 +1,10 @@
-DB_USER=<db-username>
+DB_USER=process.env.TEST_DB_USER
```
Comment (chidioguejiofor) [inline .env-sample]: change this to 

```suggestion
DB_USER=<db-username>
```

**Review by chidioguejiofor — CHANGES_REQUESTED** [review]
Nice work! Your PR title said you integrated coveralls but I couldn't find anything related here. I think you should update the title. I also think you should add the CircleCI badge to the ReadME so that folks would know when our build is passing or failing. 

Made a couple other comments in code and you need to rebase (merge conflicts)

File: .circleci/config.yml (hunk 1 above)
Comment (Runo-saduwa) [reply .circleci/config.yml]: Okay

File: .env-sample (hunk 2 above)
Comment (Runo-saduwa) [reply .env-sample]: Ok, i'm on it

**Review by chidioguejiofor — APPROVED** [review]
Looks good

**Pull request merged**